sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -100,41 +94,39 @@ enum class use {
 }
 ```
 
+#### Shape
+Size of a `joint_matrix` that refers to its number of rows `Rows` and number of columns `Cols` must be constant.
+
 #### Layout
-Besides row major and column major layouts, `layout` is flexible enough to introduce custom layouts such as packed layout.
+This specifies the memory layout and it can be row major and column major layouts
 
 ```c++
 namespace sycl::ext::oneapi::experimental::matrix {
 enum class layout {
   row_major,
   col_major,
-  packed,
   dynamic
-};
+ };
 }
 ```
 
 
 #### Memory Scope
-In this experimental API version, we used the terminology of `joint_matrix` instead of plain `matrix` to emphasize that the matrix is shared among a group of work items and is not private to each work item. The memory scope is added as an additional template parameter and is also part of the constructor arguments.
+In this API, we use the terminology of `joint_matrix` instead of plain `matrix` to emphasize that the matrix is shared among a group of work items and is not private to each work item. The memory scope is added as an additional template parameter and is also part of the constructor arguments.
```

Comment:
```suggestion
In this API, we use the terminology of `joint_matrix` instead of plain `matrix` to emphasize that the matrix is shared among a group of work items and is not private to each work item. The group scope is added as an additional template parameter and is also part of the constructor arguments.
```